libs/@local/status/crate/rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly-2023-03-06"
```

Comment:
Could you add a configuration to `.github/renovate.json` as well, please?

Reply:
[Add hash-status crate to renovate config](https://github.com/hashintel/hash/pull/1787/commits/02fa9742f92c9538612f297cb1753fce393011b1)

I think that's correct?

Reply:
Looks good, thanks. I will most likely need to adjust the file in the next days anyway because the group does not seem to work as expected, but having the configuration as it is now in place is easier to pick this up as well.